### PR TITLE
Update for 0.211.7

### PR DESCRIPTION
**Pull request by healiha**
Update code base and increase version number for new release.

### CODE REVIEW

File: ValheimPlus/GameClasses/ZNet.cs [hunk 1]
```diff
@@ -66,6 +50,28 @@ private static void Postfix(ref ZNet __instance)
...
+            for (int i = 0; i < il.Count; i++)
+            {
+                if (il[i].Calls(method_ZNet_GetNrOfPlayers))
+                {
+                    il[i + 1].operand = Configuration.Current.Server.maxPlayers;
+                    return il.AsEnumerable();
+                }
+            }
+
+            ZLog.LogError("Failed to apply ZNet.RPC_PeerInfo.Transpiler");
```
Comment (increddibelly) [inline ValheimPlus/GameClasses/ZNet.cs]: this is true, but it's very technical. 
```suggestion
            ZLog.LogError("Failed to alter server player limit (ZNet.RPC_PeerInfo.Transpiler)");
```

**Review by increddibelly — DISMISSED** [review]
minor suggestion. LGTM

**Review by smietanka — DISMISSED** [review]

**Comment by mikael0hlsson** [comment]
Lets get it out :)

**Comment by healiha** [comment]
As much as I appreciate people approving my PR, only @nxPublic can merge it

He should be back today, you can expect an official release of 0.9.9.9 soon.

**Comment by rodpedrassi** [comment]
there is any way to contact @nxPublic ?

**Comment by Shigbeard** [comment]
> there is any way to contact @nxPublic ?

He's currently unable to merge the pull request due to his IRL commitments. There's a relevant announcement about this in the Discord dating back to the 29th of September. All we can do is be patient.

**Comment by increddibelly** [comment]
we're on 211.9 now, I haven't checked yet if this breaks anything.
https://valheim.com/news/patch-0-211-9/

**Comment by ghost** [comment]
Why can't @nxPublic authorize someone else to merge, if he is unable to attend to this project?

**Review by nxPublic — APPROVED** [review]
❤️  Thank you @healiha 

Sorry at everyone having waited so long for me to merge everything. 
I've already made steps to make sure it won't happen again together with the rest of the people actively curating the project!

**Comment by Shigbeard** [comment]
... was just about to comment about how Healiha told me there would be an internal discussion to deal with scenarios like this moving forward.... lmao

Cheers for this nx!

**Comment by Shigbeard** [comment]
> we're on 211.9 now, I haven't checked yet if this breaks anything.
> https://valheim.com/news/patch-0-211-9/

this patch still works with 211.9

**Comment by ghost** [comment]
Thanks @nxPublic for merging - is someone gonna create a release?

**Pull request merged**